tests/test_models/test_engine/test_db_storage.py

```diff
@@ -68,21 +64,52 @@ def test_dbs_func_docstrings(self):
                             "{:s} method needs a docstring".format(func[0]))
 
 
-class TestFileStorage(unittest.TestCase):
+class TestDBStorage(unittest.TestCase):
     """Test the FileStorage class"""
-    @unittest.skipIf(models.storage_t != 'db', "not testing db storage")
+    @unittest.skipIf(models.storage_t != 'db', "testing db storage")
```

Comment:
Unify style to quote strings, use only one of both, simple quotes or double quotes